src/middlewared/middlewared/plugins/system_vendor/vendor.py

```diff
@@ -11,13 +11,16 @@
 
 class VendorService(Service):
 
+    class Config:
+        namespace = 'system.vendor'
+        cli_private = True
+
     @api_method(VendorNameArgs, VendorNameResult, private=True)
     def name(self) -> str | None:
         try:
             with open(SENTINEL_FILE_PATH, 'r') as file:
-                if contents := file.read():
-                    return json.loads(contents).get('name')
-        except (FileNotFoundError, json.JSONDecodeError):
```

Comment:
This was left over from my previous pr. It was originally there because there was going to be some logic if the file was empty, but it was decided to never raise any exceptions in `name`. My change cuts out the extra `read()` step.

Reply:
So it is always better that we have explicit exceptions mentioned in the except clause and not leave it broad imho, so whichever exceptions you think can be raised i think can be listed down here